templates/probot/files/LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License
+
+Copyright (c) 2020, chadcarlson <chad.carlson@platform.sh>
```

Comment:
Wait, what?  Why do we need to add a custom license file?

(And anything we release should be MIT, as that has broadest compatibility.)

Reply:
I think that's just the default. Will change.